index.js

```diff
@@ -14,9 +14,16 @@ const getResult = (regex, aInput, output) => {
 }
 
 const getPutsToConsoleLog = (aInput) => {
-  const regex = /^(\s*)puts (.*)$/g;
+  const regex = /^puts (.*)$/g;
   return getResult(regex, aInput, (match) => `${match[1]}console.log(${match[2]})`);
 }
+
+
+const getAllToEvery = (aInput) => {
+  const regex = /^(\s*)(.*).all\?$/g;
```

Comment:
Hi @impana-srinivas , the all? function in ruby requires a function to be passed into it: for example: 
Ruby: ['ant', 'bear', 'cat'].all? { |word| word.length >= 3 }                    #=> true
JS:      ['ant', 'bear', 'cat'].every(word => word.length >= 3)               #=> true
I haven't done the higher order functions in the brackets, but I will combine them with this in the future. Thanks